src/main/kotlin/com/grepp/nbe1_3_team04/chat/domain/Chat.kt

```diff
@@ -0,0 +1,75 @@
+package com.grepp.nbe1_3_team04.chat.domain
+
+import com.grepp.nbe1_3_team04.global.domain.BaseEntity
+import com.grepp.nbe1_3_team04.member.domain.Member
+import jakarta.persistence.*
+import jakarta.validation.constraints.NotNull
+import org.hibernate.annotations.SQLDelete
+import java.io.Serial
+import java.io.Serializable
+
+@Entity
+@SQLDelete(sql = "UPDATE chat SET is_deleted = 'true' WHERE chat_id = ?")
+class Chat(
+    chatroom: Chatroom,
+    member: Member,
+    chatType: ChatType,
+    text: String
+) : BaseEntity(), Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var chatId: Long? = null
```

Comment:
Id는 val로 변경 불가능하도록 하는게 좋지 않을까요?
```suggestion
    val chatId: Long? = null
```

Reply:
간단하게 테스트 진행해봤는데 val로 두어도 잘 되네요!

그럼 set을 완전 차단하는게 훨씬 좋아보여요

id 값은 val로 통일하는거로 하죠!

![image](https://github.com/user-attachments/assets/e7591fea-21b5-485d-a73e-87a3c3ebe001)